go/vt/vttest/vtprocess.go

```diff
@@ -223,6 +223,9 @@ func VtcomboProcess(environment Environment, args *Config, mysql MySQLManager) (
 		"--dbddl_plugin", "vttest",
 		"--foreign_key_mode", args.ForeignKeyMode,
 		"--planner-version", args.PlannerVersion,
+
+		fmt.Sprintf("--tablet_refresh_interval=%v", args.TopoTabletRefreshInterval),
+		fmt.Sprintf("--leader_check_interval=%s", "3s"),
```

Comment:
Should this be `5` instead of `3`?

```
leaderCheckInterval     = 5 * time.Second
```

Reply:
for vttestserver 5 seems to be high value. I order to make vttestserver fast, we can use 3 instead...